packages/hardhat/.env.example

```diff
@@ -0,0 +1,2 @@
+ALCHEMY_API_KEY=""
+DEPLOYER_PRIVATE_KEY=""
```

Comment:
Maybe we could a line on the README to tell people to copy this to `.env` and fill with their keys?